docs/releases.md

```diff
@@ -0,0 +1,60 @@
+# Releases
+
+Vite releases follow [Semantic Versioning](https://semver.org/). You can see the latest stable version of Vite in the [Vite npm package page](https://www.npmjs.com/package/vite).
+
+A full changelog of past releases is [available on GitHub](https://github.com/vitejs/vite/blob/main/packages/vite/CHANGELOG.md).
+
+::: tip note
+The next Vite Major release will happen after the Node 16 EOL in September.
+
+Check out the [Vite 5 discussion](https://github.com/vitejs/vite/discussions/12466) for more information.
+:::
+
+## Release Cycle​
+
+Vite does not have a fixed release cycle.
+
+- **Patch** releases are released as needed.
+- **Minor** releases always contain new features, with a typical time frame of ~2-3 months in between. Minor releases always go through a beta pre-release phase.
```

Comment:
Maybe we should remove the time frame here, I think we should cut minors whenever we need to introduce a new feature.

```suggestion
- **Minor** releases always contain new features and are also released as needed. Minor releases always go through a beta pre-release phase.
```